scripts/terraform.sh

```diff
@@ -23,6 +23,31 @@ if [[ -z "${PROJECT_ID:-}" ]]; then
   echo "✋ PROJECT_ID must be set"
 fi
 
+# Ensure not running on prod resources
+readonly PROTECTED_PROJECT_IDS=(
```

Comment:
Should we do the inverse and have an allow list instead?

Reply:
That would be much safer, but also means a larger list to maintain

Reply:
👍